packages/base/src/widget.ts

```diff
@@ -175,7 +177,7 @@ export class JupyterCadPanel extends SplitPanel {
   }
 
   get cameraSettings(): CameraSettings | undefined {
```

Comment:
Let's change it to `CameraSettings` everywhere and be sure it's defined?

```suggestion
  get cameraSettings(): CameraSettings {
```

Reply:
This will eventually also clean your command a bit, removing some checks

Reply:
Ok thanks. And the same changed applies for the setter I guess?

Reply:
Yep :) And TypeScript will then ask you to change also the interface an more places I suppose. It's good cleanup 👍🏽